pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.towncrier]
+    package = "synapse"
+    filename = "CHANGES.md"
```

Comment:
```suggestion
    filename = "CHANGELOG.md"
```

Maybe we want to rename this file in a separate PR to match Synapse actually.

Reply:
> Maybe we want to rename this file in a separate PR to match Synapse actually.

Not sure it's worth the faff, the confusion risk is pretty low.